CMSgov/design-system
# [WNMGDS-2226] Update privacy settings content

## Summary

https://jira.cms.gov/browse/WNMGDS-2226

- Updated English and Spanish content

## How to test

1. Open Storybook and navigate to the HealthCare.gov Footer story
2. Click the "Privacy settings" button
    - Content should be updated in English and Spanish
    - Links in the dialog should have normal link styling (fixed bug) 
3. Also try the core PrivacySettingsDialog story

## Checklist




### If this is a change to code:
